README.md

```diff
@@ -1 +1,107 @@
+# Check Plugin Template
+
+## Overview
+check-plugin-template is a template repository which wraps the [Sensu Plugin SDK][2].
+To use this project as a template, click the "Use this template" button from the main project page.
+Once the repository is created from this template, you can use the [Sensu Plugin Tool][9] to
+populate the templated fields with the proper values.
+
+## Functionality
+
+After successfully creating a project from this template, update the `checkOptions` struct in
+addition to the `executeCheck` and `checkArgs` functions in [main.go][7] to customize the behavior
+of this check plugin.
+
+When writing or updating a plugin's README from this template, review the Sensu Community
+[plugin README style guide][3] for content suggestions and guidance. Remove everything
+prior to `# {{ .Name }}` from the generated README file, and add additional context about the
+plugin per the style guide.
+
+## Releases with Github Actions
+
+To release a version of your project, simply tag the target sha with a semver release without a `v`
+prefix (ex. `1.0.0`). This will trigger the [GitHub action][5] workflow to [build and release][4]
+the plugin with goreleaser. Register the asset with [Bonsai][8] to share it with the community!
+
+***
+
+# {{ .Name }}
+
+## Table of Contents
+- [Overview](#overview)
+- [Files](#files)
+- [Usage examples](#usage-examples)
+- [Configuration](#configuration)
+  - [Asset registration](#asset-registration)
+  - [Resource definition](#resource-definition)
+- [Installation from source](#installation-from-source)
+- [Additional notes](#additional-notes)
+- [Contributing](#contributing)
+
+## Overview
+
+The {{ .Name }} is a [Sensu Check][6] that ...
+
+## Files
+
+## Usage examples
+
+## Configuration
+
+### Asset registration
+
+[Sensu Assets][10] are the best way to make use of this plugin. If you're not using an asset, please
+consider doing so! If you're using sensuctl 5.13 with Sensu Backend 5.13 or later, you can use the
+following command to add the asset:
+
+```
+sensuctl asset add {{ .GithubUser }}/{{ .GithubProject }}
+```
+
+If you're using an earlier version of sensuctl, you can find the asset on the [Bonsai Asset Index][https://bonsai.sensu.io/assets/{{ .GithubUser }}/{{ .GithubProject }}].
+
+### Resource definition
+
+```yml
+---
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: {{ .GithubProject }}
+  namespace: default
+spec:
+  command: {{ .GithubProject }} --example example_arg
+  subscriptions:
+  - system
+  runtime_assets:
+  - {{ .GithubProject }}
+```
+
+## Installation from source
+
+The preferred way of installing and deploying this plugin is to use it as an Asset. If you would
+like to compile and install the plugin from source or contribute to it, download the latest version
+or create an executable script from this source.
+
+From the local path of the {{ .GithubProject }} repository:
+
+```
+go build -o /usr/local/bin/{{ .GithubProject }} main.go
```

Comment:
See [my comment on `sensuctl-plugin-template`](https://github.com/sensu-community/sensuctl-plugin-template/pull/2/files#r377258854)

Reply:
https://github.com/sensu-community/handler-plugin-template/pull/4#discussion_r377097932

Reply:
I think we should nip this in the bud before people start (hopefully) using those templates, but I don't want to block this PR.

Here's the course of action I suggest: I'll open issues against the plugin style guide and the existing plugins that it references. If people agree with my proposed changes, I can then open appropriate PRs across the board.